src/components/Profile/components/PersonalInfoList/index.tsx

```diff
@@ -78,52 +76,62 @@ const PersonalInfoList = ({ myProf, profile, isOnline, createSnackbar }: Props)
   const [advisorsList, setAdvisorsList] = useState<AdvisorType[]>([]);
   const [showMailCombo, setShowMailCombo] = useState(false);
   const [graduationInfo, setGraduationInfo] = useState<Graduation | null>(null);
-  const isStudent = profile.PersonType?.includes('stu');
-  const isFacStaff = profile.PersonType?.includes('fac');
-  const isAlumni = profile.PersonType?.includes('alu');
+  const isStudent = checkIsStudent(profile); //profile.PersonType.includes('stu');
+  const isFacStaff = checkIsFacStaff(profile); //profile.PersonType.includes('fac');
+  const isAlumni = checkIsAlumni(profile); //profile.PersonType.includes('alu');
```

Comment:
```suggestion
  const isStudent = checkIsStudent(profile);
  const isFacStaff = checkIsFacStaff(profile);
  const isAlumni = checkIsAlumni(profile);
```
Better to delete the old code